docs/pages/architecture/nodes.mdx

```diff
@@ -1,53 +1,60 @@
 ---
```

Comment:
As a general comment, we ought to add a page similar to this which explains exactly what a Teleport Agent is, what it does and when you should use it. At the moment it's easy to see what the architecture of `ssh_service` is, but it's less clear for any of the other protocols we support.